code-samples/eventing/bookstore-sample-app/frontend/client/components/CommentDisplay.js

```diff
@@ -0,0 +1,22 @@
+const CommentDisplay = ({ comment }) => {
+	// Assume receiving a comment object
+	return (
+		<div className='flex my-4 p-4 justify-center'>
+			{/* Customized darkmode not working as expected yet */}
+			<div className='comment-display w-full w-7/12 flex flex-row rounded-lg p-4 bg-gray-800 text-white dark:bg-white'>
+				<div className='flex items-center justify-center md:justify-start md:w-1/12'>
+					<img
+						src={comment.avatar}
+						alt='Avatar'
+						className='rounded-full w-10 h-10'
+					/>
```

Comment:
![CleanShot 2024-04-01 at 23 40 56@2x](https://github.com/knative/docs/assets/36619969/6e38cff2-db53-4fd5-9627-4ab821bb76a4)

It seems like the avatar is not displayed properly on my end. And we can choose something like this as the fixed avatar. What do you think?

![user](https://github.com/knative/docs/assets/36619969/533c0f33-cc6f-42f9-bc55-5b5de84eb9b9)